xbmc/addons/kodi-addon-dev-kit/src/api2/kodi-addon-sharedlibrary/kodi-addon-sharedlibrary-api2-config.cmake.in

```diff
@@ -0,0 +1,35 @@
+# - kodi-addon-sharedlibrary config mode
+#
+# Defines the following variables:
+#  kodi-addon-sharedlibrary_FOUND        - true
+#  kodi-addon-sharedlibrary_API_LEVEL    - detected api level
+#  kodi-addon-sharedlibrary_VERSION      - version of the dev kit library found, e.g. 2.0.0
+#  kodi-addon-sharedlibrary_INCLUDE_DIRS - header directories with which to compile
+#  kodi-addon-sharedlibrary_LINKER_FLAGS - flags that must be passed to the linker
+#  kodi-addon-sharedlibrary_LIBRARIES    - names of the libraries with which to link
+#  kodi-addon-sharedlibrary_LIBRARY_DIRS - directories in which the libraries are situated
+#
+# propagate these properties from one build system to the other
+set (kodi-addon-sharedlibrary_API_LEVEL "2")
+set (kodi-addon-sharedlibrary_VERSION "2.0.0")
+set (kodi-addon-sharedlibrary_INCLUDE_DIRS @prefix@/include)
+set (kodi-addon-sharedlibrary_LIBRARY_DIRS "")
+set (kodi-addon-sharedlibrary_LINKER_FLAGS "")
+set (kodi-addon-sharedlibrary_CONFIG_VARS "")
+set (exec_prefix @exec_prefix@)
```

Comment:
By the values here I'm not sure how I can set it more easy. 

With cmake alone it was easy and can be defined by the `@...@` but with support also to the old configure has I not found a way to allow several versions of them in future.